scene/gui/graph_edit.cpp

```diff
@@ -989,10 +989,10 @@ void GraphEdit::_top_layer_draw() {
 	_update_scroll();
 
 	if (connecting) {
-		Node *fromn = get_node(NodePath(connecting_from));
-		ERR_FAIL_COND(!fromn);
+		Node *fromn = get_node_or_null(NodePath(connecting_from));
+		ERR_FAIL_NULL_MSG(fromn, "GraphEdit: Path to node is invalid: " + connecting_from + ".");
```

Comment:
I'm unsure what to call the path here, if someone has a suggestion to make the error clearer as to what it points to please suggest

Reply:
That said, you don't have to remove it now. But I have the same question about adding the node type as a message prefix as with Area3D.

Reply:
Gotcha! Will restore to null check